nuage-tripleo-heat-templates/environments/neutron-nuage-config.yaml

```diff
@@ -0,0 +1,45 @@
+# A Heat environment file which can be used to enable a
+# a Neutron Nuage backend on the controller, configured via puppet
+resource_registry:
+  OS::TripleO::Services::NeutronDhcpAgent: OS::Heat::None
+  OS::TripleO::Services::NeutronL3Agent: OS::Heat::None
+  OS::TripleO::Services::NeutronMetadataAgent: OS::Heat::None
+  OS::TripleO::Services::NeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronOvsAgent: OS::Heat::None
+  # Override the NeutronCorePlugin to use Nuage
+
+  OS::TripleO::Docker::NeutronMl2PluginBase: /home/stack/nuage-tripleo-heat-templates/puppet/services/neutron-plugin-ml2-nuage.yaml
```

Comment:
Sunny, instead of absolute path, lets use relative path. Because all Nuage related templates will be part of same directory right and also that will give us flexibility to change the path of this directory if needed.